packages/ERTP/CONTRIBUTING.md

```diff
@@ -4,44 +4,42 @@ Thank you!
 
 ## Contact
 
-We use github issues for all bug reports: https://github.com/Agoric/ERTP/issues
+We use github issues for all bug reports:
+https://github.com/Agoric/agoric-sdk/issues Please add an [ERTP]
+prefix to the title and ERTP tag to ERTP-related issues.
 
 ## Installing, Testing
 
 You'll need Node.js version 11 or higher. 
 
-* git clone https://github.com/Agoric/ERTP/
-* npm install
-* npm test
+* git clone https://github.com/Agoric/agoric-sdk/
+* cd packages/ERTP
+* yarn install
+* yarn test
 
 ## Pull Requests
 
 Before submitting a pull request, please:
 
-* run `npm test` and make sure all the unit tests pass
-* run `npm run lint-fix` to reformat the code according to our
+* run `yarn test` and make sure all the unit tests pass
+* run `yarn run lint-fix` to reformat the code according to our
```

Comment:
Also, I consolidated our `eslint`/`prettier` dependencies in the top level `package.json`, so I think you *must* do `yarn install` from the top level to be able to run `lint-fix` from the individual packages.

(I figured this was reasonable because the tarballs we publish to NPM aren't supposed to contain enough to do lint checks.. they might not even contain all the original source code, just some rollup-ed `dist/` output artifact)

Reply:
By consolidate, do you mean removing prettier from the dependencies list in packages, such as `packages/zoe/package.json`?

Reply:
Correct. That also gave me an opportunity to unify the versions of prettier/eslint and their plugins, so all packages are now using a consistent set of tools.

I think the next step is to consolidate the configuration files as well, but I'm not exactly sure how eslint/prettier works, and whether they will climb upwards to find an `.eslintrc` in the monorepo root, or if they want to find it in the package root.